pinot-query-planner/src/main/java/org/apache/calcite/sql/fun/PinotOperatorTable.java

```diff
@@ -248,4 +316,16 @@ private boolean isPinotAggregationFunction(String name) {
     }
     return aggFunctionType != null && !aggFunctionType.isNativeCalciteAggregationFunctionType();
   }
+
+  private boolean isPinotTransformFunction(String name) {
+    TransformFunctionType transformFunctionType = null;
+    if (isTransformFunctionRegisteredWithOperatorTable(name)) {
+      try {
+        transformFunctionType = TransformFunctionType.valueOf(name);
+      } catch (IllegalArgumentException e) {
+        // Ignore
```

Comment:
Any particular reason for ignoring this exception?

Reply:
Expect null if function is not recognized.

Reply:
this error shouldn't occur if the isTransform... method is correctly capture the name